data/json/items/gun/308.json

```diff
@@ -446,5 +455,48 @@
     "min_cycle_recoil": 2700,
     "pocket_data": [ { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "ar10mag_20rd", "ar10_makeshiftmag" ] } ],
     "melee_damage": { "bash": 12 }
+  },
+  {
+    "id": "rfb_308",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "7.62x51mm bullpup rifle" },
+    "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines of the heavy automatic rifle and a 7.62x51mm cartridge.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rfb_308",
+        "name": { "str_sp": "Kel-Tec RFB" },
+        "description": "This is a semi-automatic bullpup rifle with a lightweight, polymer frame, using magazines from the FN FAL and the .308 Winchester cartridge.  The rifle, rorward-ejection, bullpup (abbreviated as RFB) features forward case ejection, ambidextrous controls, and structural use of the barrel, uncommon in modern designs.",
+        "weight": 1
+      }
+    ],
+    "weight": "3900 g",
+    "volume": "3073 ml",
+    "longest_side": "701 mm",
+    "barrel_length": "470 mm",
+    "price": 180000,
+    "price_postapoc": 4000,
+    "to_hit": -2,
+    "material": [ "steel", "aluminum", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "308" ],
+    "dispersion": 500,
```

Comment:
Same here. Please add sight_dispersion in addition to your original dispersion value.

Reply:
Fixed in both rifles